src/features/game/toast/ToastManager.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { Panel } from "components/ui/Panel";
+import { ToastContext } from './ToastQueueProvider';
+
+export const ToastManager = () => {
+  const { toastList, removeToast } = useContext(ToastContext);
+  const [listed, setListed] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (toastList.length >= 1) {
+      setListed(true);
+    } else {
+      setListed(false);
+    }
+  }, [toastList]);
+
+  return (
+    <div>
+      {listed &&
+        <div className="bg-brown-600 p-0.5 text-white shadow-lg flex flex-col items-end mr-2 sm:block fixed top-20 left-2 z-[99999]">
```

Comment:
<img width="248" alt="Screen Shot 2022-01-30 at 7 28 45 pm" src="https://user-images.githubusercontent.com/11745561/151720559-eba03b17-76f9-4048-a5db-45b18b48d974.png">


One small UI suggestion, add `text-shadow` to make the text more readable. Also consider removing the brown background since the panel already has the brown background 👍